testrail/client.py

```diff
@@ -172,13 +183,23 @@ def active_plans(self):
     # Run Methods
     @methdispatch
     def runs(self):
-        return map(Run, self.api.runs(self._project_id))
+        return RunContainer(list(map(Run, self.api.runs(self._project_id))))
 
     @runs.register(Milestone)
     def _runs_for_milestone(self, obj):
         return RunContainer(filter(
             lambda r: r.milestone.id == obj.id, self.runs()))
 
+    @runs.register(Active)
+    def _runs_for_active(self, obj):
```

Comment:
I found that the API will return a maximum of 250 objects per call, with no pagination support that I could locate. I was in a scenario where I needed to close 250 runs out of roughly 400 total runs, but the API wouldn't return the last 100 active runs. The ultimate take away was that we definitely should support adding params to the API call to filter results, and not rely solely on the container for filtering.

After thinking about it some more, maybe a better approach would be to create a Filter class, which you then instantiate, configure, and then pass to the runs/milestones/plans/etc calls?

pseudo code:
```
filter = tr.Filter()
filter.start_date  = datetime.datetime.now() - timedelta(day=20)
filter.end_date = datetime.datetime.now() 
tr.runs(filter)
```

How does something like that sound?